### PR TITLE
chore(node): move genesis to runtime

### PR DESCRIPTION
**Pull request by enddynayn**
Removed the genesis builders for Paseo and Frequency, as these are now referenced directly from the generated spec file.

Consolidated Frequency-local and Frequency-no-relay into a single genesis_builder function for improved clarity and maintainability.

### CODE REVIEW

File: .github/workflows/verify-pr-commit.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ on:
     branches:
       - main
       - "*-development" # Feature Branches should suffix with -development
+      - "chore/mv-development-genesis-to-runtime-issue-*"
```
Comment (enddynayn) [inline .github/workflows/verify-pr-commit.yml:11]: rm

File: runtime/frequency/src/genesis/presets.rs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+		Default::default(),
+		2091.into(),
+	)
+}
+
+/// Provides the JSON representation of predefined genesis config for given `id`.
+pub fn get_preset(id: &sp_genesis_builder::PresetId) -> Option<sp_std::vec::Vec<u8>> {
+	let genesis = match id.try_into() {
+		Ok("development") => development_genesis_config(),
+		Ok("paseo") => paseo_genesis_config(),
```
Comment (enddynayn) [inline runtime/frequency/src/genesis/presets.rs:49]: Add flags here.

File: runtime/frequency/src/apis.rs [hunk 3]
```diff
@@ -152,12 +149,14 @@ impl_runtime_apis! {
...
 
 		fn get_preset(id: &Option<sp_genesis_builder::PresetId>) -> Option<Vec<u8>> {
-			get_preset::<RuntimeGenesisConfig>(id,  &genesis_config_presets::get_preset)
+			get_preset::<RuntimeGenesisConfig>(id,  &crate::genesis::presets::get_preset)
 		}
 
 		fn preset_names() -> Vec<sp_genesis_builder::PresetId> {
 			vec![
 				sp_genesis_builder::PresetId::from("development"),
+				sp_genesis_builder::PresetId::from("paseo"),
```
Comment (wilwade) [inline runtime/frequency/src/apis.rs:158]:
```suggestion
				sp_genesis_builder::PresetId::from("frequency-local"), // Local Paseo with paraId 2000
				sp_genesis_builder::PresetId::from("frequency-paseo"), // Testnet Paseo with paraid 4000
```

File: runtime/frequency/src/genesis/frequency.rs [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+use sp_core::ByteArray;
+#[cfg(not(feature = "std"))]
+use sp_std::alloc::format;
+use sp_std::vec;
+
+use common_runtime::constants::currency::DOLLARS;
+
+use sp_runtime::traits::AccountIdConversion;
+
+pub mod frequency_mainnet_keys {
```
Comment (wilwade) [inline runtime/frequency/src/genesis/frequency.rs:11]: Do we still need this file for benchmarking? If not we can delete

File: runtime/frequency/src/apis.rs [hunk 5]
```diff
@@ -152,12 +149,14 @@ impl_runtime_apis! {
...
+				sp_genesis_builder::PresetId::from("frequency"),
```
Comment (wilwade) [inline runtime/frequency/src/apis.rs:159]: For `frequency` and `frequency-testnet` can just return the parsed JSON spec files.

For `frequency-no-relay` and `frequency-local` the generated ones are good.

File: runtime/frequency/src/genesis/paseo.rs
Comment (wilwade) [inline runtime/frequency/src/genesis/paseo.rs:1]: This is `frequency-local`

Comment (wilwade) [reply runtime/frequency/src/genesis/paseo.rs:1]: Or make it "development-defaults"?

File: runtime/frequency/src/genesis/presets.rs [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+		paseo_endowed_accounts(),
+		paseo_session_keys(),
+		paseo_council_members(),
+		paseo_technical_committee_members(),
+		load_genesis_schemas(),
+		2000.into(),
+	)
+}
+
+fn frequency_genesis_config() -> serde_json::Value {
```
Comment (wilwade) [inline runtime/frequency/src/genesis/presets.rs:32]: We can remove IF it is not used by benchmarks

**Review by JoeCap08055 — COMMENTED** [review]
@enddynayn not sure what I'm supposed to be seeing, but I pulled & built this branch and I don't see the schemas that are supposed to be in the genesis block.

**Pull request merged**